hieradata/integration.yaml

```diff
@@ -39,6 +39,10 @@ govuk_cdnlogs::monitoring_enabled: false
 govuk_crawler::seed_enable: true
 
 govuk_elasticsearch::dump::run_es_dump_hour: '9'
+govuk_elasticsearch::backup::s3_bucket: 'kibana-dashboards'
```

Comment:
on naming: maybe we could name these things after the cluster itself, rather than the index?